boards/arm/stm32f746g_disco/pinmux.c

```diff
@@ -22,9 +22,9 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PC6, STM32F7_PINMUX_FUNC_PC6_USART6_TX},
 	{STM32_PIN_PC7, STM32F7_PINMUX_FUNC_PC7_USART6_RX},
 #endif
-#ifdef CONFIG_PWM_STM32_3
+#if DT_HAS_NODE(DT_NODELABEL(pwm2))
```

Comment:
pwm3

Reply:
This being said, in device tree only pwm1 is enabled...

Reply:
Fixed. Looks like pwm3 should be the enabled node. I can add a separate commit to fix that, even though it is out of the scope of this PR.